tests/v2/validation/rbac/rbac_test.go

```diff
@@ -269,16 +282,26 @@ func (rb *RBTestSuite) ValidateRemoveClusterRoles() {
 
 	err := users.RemoveClusterRoleFromUser(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
+
+	log.Info("Validating Cluster Role Template Binding is deleted for the user.")
+	crtbs, err := rb.client.Steve.SteveType("management.cattle.io.clusterroletemplatebinding").List(nil)
```

Comment:
This list call can use the query, to get a specific role, and if it's not there, then report a failures.